lib/dealership.rb

```diff
@@ -0,0 +1,57 @@
+require './lib/car'
+
+class Dealership
+
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory.push(car)
+    return nil
```

Comment:
I think it would be okay if you returned the car here rather than nil. Not necessary, just another option